### DISCUSSION TITLE
how to set up global powers for Joomla Component builder

### DISCUSSION THREAD

**Discussion by schabert68** [joomengine/Joomla-Component-Builder#974] [Extention Issues] [answered]
I have succeeded in installing JCP on Joomla 5.2. However, I don't know where to start.

Do I need to setup global powers to be able to build a component for my website, if so, how do I do it? do I need to get the pro version too be able to use jcb and compile the components for my site?

**Comment by Llewellynvdm** [comment, marked as answer]
You should be able to build a component with JCB and not need any understanding of power, since JCB will automatically add what is needed.

With all tools of this **vast nature** starting with the tutorials is the right place: https://www.youtube.com/playlist?list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE

While these tutorial are old, they are still enough, and accurate to get you going and help you advance in the understanding on how JCB works.
